content/References/ConceptsAndTerms.md

```diff
@@ -20,6 +20,8 @@ While TrueNAS is designed for and ever-evolving towards increased user friendlin
 
 * Software as a Service (SaaS): Software as a service (SaaS) is a software licensing and delivery model in which software is licensed on a subscription basis and is centrally hosted. It is sometimes referred to as "on-demand software".
 
+* SSD Metadata: Filesystem metadata that can be stored on SDs for faster performance while large data files can be stored on HDDs for lower costs.
+
 * Storage: Computer data storage is a technology consisting of computer components and recording media that are used to retain digital data.
 
 * File System: A file system or filesystem (fs) controls how data is stored and retrieved. Without a file system, data placed in a storage medium would be one large body of data with no way to tell where one piece of data stops and the next begins.
```

Comment:
filesystem is only correct when specifically talking about the command filesystem or if it is typed this way in the UI and you are specifically talking about that command or UI element.
```suggestion
* File System: A file system controls how data is stored and retrieved. Without a file system, data placed in a storage medium would be one large body of data with no way to tell where one piece of data stops and the next begins.
```